fossology/fossology
# feat(licenseref): add new exception text to fossology database

## Description

Add licenses exception texts to license_ref.

### Changes

License text has been added for the following licenses exceptions. 

*  Bison-exception-2.2
*  GCC-exception-3.1
*  Autoconf-exception-2.0
*  GCC-exception-2.0
*  Font-exception-2.0
*  389-exception
*  Autoconf-exception-3.0
*  Bootloader-exception
*  eCos-exception-2.0
*  CLISP-exception-2.0
*  Classpath-exception-2.0
*  DigiRule-FOSS-exception
*  FLTK-exception
*  freertos-exception-2.0
*  gnu-javamail-exception
*  i2p-gpl-java-exception
*  PS-or-PDF-font-exception-20170817
*  Libtool-exception
*  LLVM-exception
*  LZMA-exception
*  mif-exception
*  OCaml-LGPL-linking-exception
*  OCCT-exception-1.0
*  OpenJDK-assembly-exception-1.0
*  openvpn-openssl-exception
*  Qt-GPL-exception-1.0
*  Qt-LGPL-exception-1.1
*  Qwt-exception-1.0
*  u-boot-exception-2.0
*  WxWindows-exception-3.1

## How to test

Upload the files with the above license texts and  check the nomos and monk results.
